CONTRIBUTING.md

```diff
@@ -121,22 +121,23 @@ pnpm exec changeset
 
 ### Running benchmarks
 
-We have benchmarks to keep performance under control. You can run these by running (from the project root):
+We have benchmarks to keep performance under control. They are located in the `benchmarks` directory, and it exposes a CLI you can use to run them.
+
+You can run all available benchmarks sequentially by running (from the project root):
 
 ```shell
-pnpm run benchmark --filter astro
+pnpm run benchmark
 ```
 
-Which will fail if the performance has regressed by **10%** or more.
-
-To update the times cd into the `packages/astro` folder and run the following:
+To run a specific benchmark only, you can add the name of the benchmark after the command:
 
 ```shell
-node test/benchmark/build.bench.js --save
-node test/benchmark/dev.bench.js --save
+pnpm run benchmark memory
 ```
 
-Which will update the build and dev benchmarks.
+Use `pnpm run benchmark --help` to see all available options.
+
+In a PR on GitHub, you can also comment `!bench`, `!bench memory`, etc, to run the benchmarks on CI. The benchmark will run on both the PR branch and the `main` branch, and the results will be posted as a comment.
```

Comment:
Can you explain the difference between `!bench` and `!bench memory`? Maybe a list of commands like:

- `!bench` - Benchmarks rendering performance against the PR branch and main and prints out a table of each result.
- `!bench memory` - Benchmarks memory usage against the PR branch and main and prints out a table of each result.

Reply:
I was hoping the paragraphs before this already implied what they do, otherwise it would be duplicating the docs 🤔 Ideally we would expand more commands than `!bench` too so they can use `pnpm run benchmark --help` to view the full list of them.

Maybe we can rename the npm script as `pnpm run bench` so it's clearer?

Reply:
Ah ok, so the !bench command maps directly to the npm command? I didn't realize that. If so then I guess you can disregard this comment.